lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/initial-journey-selection.yaml

```diff
@@ -45,7 +45,13 @@ CHECK_EXISTING_IDENTITY:
     enhanced-verification:
       targetJourney: NEW_P2_IDENTITY
       targetState: ENHANCED_VERIFICATION
-
+    alternate-doc-invalid-dl:
+      targetJourney: INELIGIBLE
+      targetState: INELIGIBLE
+      checkFeatureFlag:
+        d02MitigationEnabled:
+          targetJourney: NEW_P2_IDENTITY
+          targetState: ALTERNATE_DOC
```

Comment:
I'm wondering if it might be better, rather than handling the feature flag in the journey map like this, to hide the emitting of this event behind the flag in the code.

With the flag disabled, this event should be treated the same as the `fail-with-ci` event. Which is complicated by the fact that we also have `fail-with-ci-and-forced-reset`.

If we ensure the code only emits the `alternate-doc-invalid-dl` event in exactly the right circumstance (when the feature flag is active), we can avoid handling it in the journey map like this, which should keep the `fail-with-ci/fail-with-ci-and-forced-reset` situation simpler.

Reply:
Journey event alternate-doc-invalid-dl is only return if mitigation feature flag is enabled.

Left the feature flag config here......as in Journey MAP this will only be displayed when mitigation flag is turned on.